### PR TITLE
Rename the TFRecords

### PR DESCRIPTION
**Pull request by piresramon**
All the TFRecords (initially distributed into three folders, were move the one folder. The dataset was renamed, and the document id is incremented from 0000 until 1215.

### CODE REVIEW

File: t5x/mc4_aux_files/mc4-total-gptimbau.train.index [hunk 1]
```diff
@@ -1,1213 +1,1213 @@
...
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0503.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0504.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0505.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0506.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0507.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0508.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0509.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0510.tfrecords
-gs://gpt-j-pt/data/tfrecords-mc4-2-gptimbau/mc4-gptimbau_0511.tfrecords
+gs://gpt-j-pt/data/tfrecords-mc4-total-gptimbau/mc4-gptimbau_0003.tfrecords
```
Comment (rodrigonogueira4) [inline t5x/mc4_aux_files/mc4-total-gptimbau.train.index:1]: nit: Since everything is now "total", maybe remove `-total`?

**Review by rodrigonogueira4 — APPROVED** [review]
Thanks for improving this! Just one minor suggestion.

File: t5x/mc4_aux_files/mc4-total-gptimbau.train.index (hunk 1 above)
Comment (piresramon) [reply t5x/mc4_aux_files/mc4-total-gptimbau.train.index:1]: Hi, Rodrigo. The problem with doing that is that the folder `tfrecords-mc4-gptimbau` with 20% (as well as `tfrecords-mc4-brwac-gptimbau` with 10% and `tfrecords-mc4-2-gptimbau` with 70%) already exists. 
If we change the folders, we will lose the data index files as they were committed in our repo: https://bitbucket.org/neuralmindteam/mesh-transformer-jax/src/1dfda38f93c8b740c7e188e1eebd8ad6f5993e5d/data/mc4-total-gptimbau.train.index?at=feature%2Fportuguese-pretraining

In other words, removing `-total` will overwrite the folder with 20%.

Comment (piresramon) [reply t5x/mc4_aux_files/mc4-total-gptimbau.train.index:1]: Another option is moving the `mc4-total-gptimbau` (100%) to another bucket into the `maritaca` project. But I am not sure if it is possible to move data between buckets of different projects.

Comment (rodrigonogueira4) [reply t5x/mc4_aux_files/mc4-total-gptimbau.train.index:1]: Like we spoke a few hours ago, lets then keep -total. Thanks.

**Review by ZanezZephyrs — APPROVED** [review]
